.github/workflows/test_stdlib_releases_with_staging.yml

```diff
@@ -0,0 +1,63 @@
+name: Test Standard Library Releases with Staging Central
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test-patch-versions:
+    name: Test Pacth Standard Library Releases
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'ballerina-platform/ballerina-distribution'
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+
+      - name: Run Standard Library Tests
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+          devCentralToken: ${{ secrets.BALLERINA_DEV_CENTRAL_ACCESS_TOKEN }}
+        run: ./gradlew clean testStdlibs --stacktrace --scan --console=plain --no-daemon --continue
+
+      - name: Run Standard Library Tests w/ Staging
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+          devCentralToken: ${{ secrets.BALLERINA_DEV_CENTRAL_ACCESS_TOKEN }}
+        run: ./gradlew testStdlibsWithStaging --stacktrace --scan --console=plain --no-daemon --continue
+
+  test-minor-versions:
+    name: Test Pacth Standard Library Releases
```

Comment:
```suggestion
    name: Test Patch Standard Library Releases
```

Reply:
Fixed in [c5dcf5c](https://github.com/ballerina-platform/ballerina-standard-library/pull/3134/commits/c5dcf5cafb944c36435e02f97db358ffc56e865c)